.changelog/4164.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Updated the **validate** command (`BC107`) to verify that the toversion property was not changed on existing Content files.
+  type: internal
+pr_number: 4164
```

Comment:
Remove this file, we don't announce new validations since the validate refactor is not officially out yet.

Reply:
Done.